app/main.py

```diff
@@ -1 +1,20 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+
+    parts = command.split()
+
+    if len(parts) == 3 and parts[0] == "mv":
+        mv, file_source, file_destination = parts
+
+        if "/" in file_destination:
```

Comment:
Rewrite the check that the value in `file_destination` is a directory or not using `os`. It will make your code platform-agnostic.